src/docfx/docset/Docset.cs

```diff
@@ -278,6 +278,11 @@ private HashSet<Document> CreateBuildScope(IEnumerable<Document> redirections)
                     }
                 }
 
+                foreach (var g in result.GroupBy(r => r.ContentType))
+                {
+                    Telemetry.TrackBuildItemCount(g.Key, g.Count());
```

Comment:
consider measure it in `BuildFiles`, the count here is not accurate if resources are dynamically scoped.